lineapy/api/api.py

```diff
@@ -389,14 +389,17 @@ def create_pipeline(
     return pipeline
 
 
-def get_function(artifact_list, input_parameters=[]) -> Callable:
+def get_function(artifact_list, input_parameters=[], use_cache=[]) -> Callable:
     """
-    Create a python function
+    Create a python function that perform the process to compute list of
```

Comment:
Too wordy, lets make our comments as declarative as possible. In this case: 

"Create a python function that computes list of ..."